.github/workflows/pr_opex_external.yml

```diff
@@ -0,0 +1,30 @@
+name: PR - Opex Dashboard External
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
```

Comment:
Add `ready_for_review`

Reply:
done in [42a1bd0](https://github.com/pagopa/io-web-profile-backend/pull/32/commits/42a1bd07a2f1b3618634d703ecbf89c979516510)